Ruby.gitignore

```diff
@@ -26,6 +27,9 @@ build/
 /vendor/bundle
 /lib/bundler/man/
 
+## Mac
+.DS_Store
```

Comment:
We're tracking this rule in `Global/OSX.gitignore` (as well as many others). Please drop this.